packaging/installer/methods/freebsd.md

```diff
@@ -4,30 +4,94 @@ title: "Install Netdata on FreeBSD"
+![image](https://user-images.githubusercontent.com/43221193/82707320-3776b800-9c52-11ea-9b8b-9ff4b1d1d9ac.png)
 
-# Install Netdata on FreeBSD
+# Install Netdata on FreeBSD ![image](https://user-images.githubusercontent.com/1153921/76029787-5fc40580-5ef2-11ea-9461-23e9049aa8f8.png)
+
+# Netdata ![](https://registry.my-netdata.io/api/v1/badge.svg?chart=web_log_nginx.requests_per_url&options=unaligned&dimensions=kickstart&group=sum&after=-3600&label=last+hour&units=installations&value_color=orange&precision=0) ![](https://registry.my-netdata.io/api/v1/badge.svg?chart=web_log_nginx.requests_per_url&options=unaligned&dimensions=kickstart&group=sum&after=-86400&label=today&units=installations&precision=0)
+
+## Install latest version
+This is how to install the latest Netdata version on FreeBSD:
+
+Install required packages (**need root permission**):
 
-## Install from ports/packages
-You can install Netdata from either the `ports` or `packages` collections. To install from packages:
 ```sh
-# pkg install netdata
+pkg install bash e2fsprogs-libuuid git curl autoconf automake pkgconf pidof Judy liblz4 libuv json-c cmake
 ```
-You also need to enable the netdata service in `/etc/rc.conf` (add `netdata_enable="YES"`) and start the service:
+
+Download Netdata:
+
 ```sh
-# service netdata start
+fetch https://github.com/netdata/netdata/releases/download/v1.22.1/netdata-v1.22.1.tar.gz
 ```
 
-## Install latest version
-This is how to install the latest Netdata version from source on FreeBSD:
+Unzip the downloaded file:
 
 ```sh
-# install required packages
-pkg install bash e2fsprogs-libuuid git curl autoconf automake pkgconf pidof Judy liblz4 libuv json-c cmake
+gunzip netdata*.tar.gz && tar xf netdata*.tar && rm -rf netdata*.tar
+```
 
-# download Netdata
-git clone https://github.com/netdata/netdata.git --depth=100
+Install Netdata in `/opt/netdata`, if you want to enable automatic updates, add `--auto-update` or `-u` for install netdata-updater in `cron` (**need root permission**):
 
-# install Netdata in /opt/netdata
-cd netdata
-./netdata-installer.sh --install /opt
+```sh
+cd netdata-v* && ./netdata-installer.sh --install /opt && cp /opt/netdata/usr/sbin/netdata-claim.sh /usr/sbin/
 ```
+
+You also need to enable the netdata service in `/etc/rc.conf`:
+
+```sh
+sysrc netdata_enable="YES"
+```
+
+Finally, and very importantly, update Netdata using the script provided by the Netdata team (**need root permission**):
+
+```sh
+cd /opt/netdata/usr/libexec/netdata/ && ./netdata-updater.sh
+```
+
+All set, we can now access the Netdata Web Interface. Ex: `http://server.ip:19999`
+
+![image](https://user-images.githubusercontent.com/2662304/48304090-fd384080-e51b-11e8-80ae-eecb03118dda.png)
+
+From Netdata v1.12 and above, anonymous usage information is collected by default and sent to Google Analytics. To read
+more about the information collected and how to opt-out, check the [anonymous statistics
+page](/docs/anonymous-statistics.md).
+
+
+# Important
```

Comment:
Please remove the `# Important`. We should have more than a single h1 `#` on a page, and the **Updating the Agent on FreeBSD** text is sufficient.